src/docs/_config.yml

```diff
@@ -2,6 +2,8 @@
 title: Brainly Style Guide
 description: Brainly Front-End Style Guide Documentation
 baseurl: "."
+keep_files:
+  - "css/main.css"
```

Comment:
Jekyll purges the output dir, this line prevents him from removing css file that is now controlled by gulp